src/scripts/process_au.py

```
@@ -0,0 +1,146 @@
+    return "GEN"
+
+def split_into_articles(text):
+    chunks = []
+    # 호주 법률 분리를 위한 정규식 패턴
+    pattern = r'((?:Section|Sec\.|§|s\.|Part|Pt\.)\s*\d+[a-zA-Z0-9\(\)\-\.]*)'
+    parts = re.split(pattern, text, flags=re.IGNORECASE)
+    
+    curr_art = None
+    
+    for p in parts:
+        p = p.strip()
```

Ruff (0.15.6)

[warning] `for` loop variable `p` overwritten by assignment target

(PLW2901)